### PR TITLE
Test: Camel case first lower case

### PR DESCRIPTION
**Pull request by jrodriguez-rc**

### CODE REVIEW

File: src/zcl_ajson_mapping.clas.testclasses.abap [hunk 1]
```diff
@@ -5,7 +5,9 @@ class ltcl_camel_case definition final for testing
   private section.
     methods:
       to_abap for testing raising zcx_ajson_error,
-      to_json for testing raising zcx_ajson_error.
+      to_json for testing raising zcx_ajson_error,
+      to_json_fist_lower for testing raising zcx_ajson_error.
```
Comment (sbcgua) [inline src/zcl_ajson_mapping.clas.testclasses.abap:9]: typo ?

**Review by sbcgua — CHANGES_REQUESTED** [review]
one typo above, the rest is fine

File: src/zcl_ajson_mapping.clas.testclasses.abap (hunk 1 above)
Comment (jrodriguez-rc) [reply src/zcl_ajson_mapping.clas.testclasses.abap:9]: What typo?

Comment (sbcgua) [reply src/zcl_ajson_mapping.clas.testclasses.abap:9]: to_json_fist_lower - to_json_first_lower :)

Comment (jrodriguez-rc) [reply src/zcl_ajson_mapping.clas.testclasses.abap:9]: Argh! I see it now, I'll fix it in the morning to let it ready to merge

**Comment by jrodriguez-rc** [comment]
> one typo above, the rest is fine

Done

**Review by sbcgua — APPROVED** [review]

**Comment by sbcgua** [comment]
Thanks ! :+1:
merging

**Pull request merged**